src/App.js

```diff
@@ -1,62 +1,20 @@
-import { Navigation } from 'react-native-navigation';
+import React, { useEffect } from 'react';
+import { Provider } from 'react-redux';
 import { sessionService } from 'redux-react-native-session';
-import isEmpty from 'lodash/isEmpty';
 
+import AppContainer from './navigators';
 import configureStore from './store/configureStore';
-import registerScreens from './screens';
-import { LOGIN_SCREEN, MAIN_SCREEN } from './constants/screens';
 
 const store = configureStore({});
-registerScreens(store);
-
-class App {
-  constructor() {
-    this.isRunning = false;
-    this.isAuthenticated = false;
 
+export default function App() {
```

Comment:
using these because functions are cheaper than arrow functions.

Reply:
🤔

Reply:
NVM, did some more digging and they are almost the same. 
The only gotcha of arrows is to avoid exporting unnamed functions, because you won't have the `displayName` (mostly for debugging)